test/cql-pytest/test_compaction_tombstone_gc.py

```diff
@@ -9,7 +9,7 @@
 # sleep to let a ttl (of `seconds`) expire and
 # the commitlog minimum gc time, in seconds,
 # to be greater than the tombstone deletion time
-def sleep_till_whole_second(seconds=1):
+def sleep_till_whole_second(seconds=2):
```

Comment:
@bhalevy could you take a look at the first commit (`cql-pytest: don't use gc_grace_seconds=0 for MVs`)?

Reply:
This patch is saddening for several reasons.
First there is the sleeps.
Second, it shows how silly it is to forbid a gc_grace_seconds=0 but allow gc_grace_seconds=1.
Thirdly, when this test was written by @bhalevy he told me, which really surprised me, that gc_grace_seconds=0 actually means wait until the next second, not zero seconds. So why do we have a problem with zero seconds.
Finally, I still don't understand why exactly we can't use gc_grace_seconds=0. Because of a bug with hints? Couldn't we fix this bug?

Reply:
Ok, I finally understand what's going on with gc_grace_seconds (my "finally" above), see my other comments. I still feel like we need to fix the underlying bug (or a really really awkward feature) instead of pretending that it no longer exists if you're not allowed to set gc_grace_seconds=0 but can still set gc_grace_seconds=1.